integration-tests/src/jvmTest/kotlin/ai/koog/integration/tests/utils/MediaTestUtils.kt

```diff
@@ -591,17 +592,14 @@ object MediaTestUtils {
     fun checkExecutorMediaResponse(response: Message.Response) {
         checkResponseBasic(response)
         val responseLowerCase = response.content.lowercase()
-        assertFalse(responseLowerCase.contains("error processing"), "Result should not contain error messages")
-        assertFalse(
-            responseLowerCase.contains("unable to process"),
-            "Result should not indicate inability to process"
-        )
-        assertFalse(responseLowerCase.contains("cannot process"), "Result should not indicate inability to process")
+        responseLowerCase shouldNotContain "error processing"
+        responseLowerCase shouldNotContain "unable to process"
+        responseLowerCase shouldNotContain "cannot process"
     }
 
     fun checkResponseBasic(response: Message.Response) {
-        assertNotNull(response, "Response should not be null")
-        assertTrue(response.content.isNotBlank(), "Result should not be empty or blank")
-        assertTrue(response.content.length > 20, "Result should contain more than 20 characters")
+        response.shouldNotBeNull()
```

Comment:
Again, you nested constructions and dedicated functions to be more kotest idiomatic:
```kt
response.shouldNotBeNull {
    content.shouldNotBeBlank()
    content shouldHaveAtLeastSize 20
}
```